alembic/versions/3474e98b321f_bug_1440471_1434425_more_tables.py

```diff
@@ -0,0 +1,44 @@
+"""bug 1440471, 1434425 remove raw_update_channels and graphics_device things
+
+This doesn't remove the graphics_device table--we're passing that to
+Django-land. This does remove the stored procedures that manipulate it.
+
+Revision ID: 3474e98b321f
+Revises: 1e188109fc6b
+Create Date: 2018-04-28 15:27:18.356710
+
+"""
+
+from alembic import op
+
+from socorro.lib.migrations import load_stored_proc
+
+
+# revision identifiers, used by Alembic.
+revision = '3474e98b321f'
+down_revision = '1e188109fc6b'
+
+
+def upgrade():
+    # Update changed stored procedures
+    load_stored_proc(op, ['backfill_matviews.sql'])
+    load_stored_proc(op, ['update_product_versions.sql'])
```

Comment:
It occurred to me that I should be reloading stored procedures before doing anything else because they often refer to things that are about to get removed. So I'm going to start doing that.